test/numbers.jl

```diff
@@ -2923,6 +2926,12 @@ immutable PR20530; end
             end
         end
     end
+    @test PR20889(2)^3 == 5
+end
+module M20889 # do we get the expected behavior without importing Base.^?
+    struct PR20889; x; end
+    ^(t::PR20889, b) = t.x + b
+    Base.Test.@test PR20889(2)^3 == 5
```

Comment:
I have to put these tests in a different module than the other tests because I'm explicitly testing for what happens when I do not import `Base.^`. Since they are in a different module, I shouldn't get `Base.Test.@test` automatically, since I haven't done `using Base.Test`.